keras_nlp/tokenizers/word_piece_tokenizer_trainer.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Trainer for Word Piece Tokenizer."""
```

Comment:
Word Piece => WordPiece. 

I don't know why, but the convention is to use it as one word with camel case.

Reply:
This is actually the conversion everywhere, for better or for worse.

https://ai.googleblog.com/2021/12/a-fast-wordpiece-tokenization-system.html

Reply:
Yep I can change it! Should we change it for everything, because the file names are named 'word_piece_tokenizer.py'. Also just to be sure when you mean camelCase its only for documentation? Since the functions `compute_word_piece_vocabulary` is in snake case so it should be `compute_wordpiece_vocabulary`?

Reply:
Keep the snake case word_piece for filenames and functions. Just when referring to it in comments and the like, it's WordPiece. Api naming all good though!

Reply:
Ah preference for me is how this was before. compute_word_piece_vocabulary

We should keep the convention that ThisCase means this_case, even if this is a weird ambiguity. In comments only call this WordPiece as per original paper

Reply:
I noticed tensorflow text used wordpiece for its naming `tensorflow_text.tools.wordpiece_vocab`, so I just decided to convert everything to wordpiece to match that